lib/src/view/settings/http_log_screen.dart

```diff
@@ -176,7 +176,11 @@ class HttpLogTile extends StatelessWidget {
       ),
       title: Text(
         httpLog.requestUrl.host == kLichessHost
-            ? httpLog.requestUrl.path
+            ? Uri(
+              path: httpLog.requestUrl.path,
+              query: httpLog.requestUrl.query,
+              fragment: httpLog.requestUrl.fragment,
```

Comment:
I see an empty `#` in each entry now. We should only show path and query and remove the fragment (which we do not even use).